indigo_app/templates/indigo_api/document/_attachments.html

```diff
@@ -58,13 +60,11 @@ <h5>or</h5>
 </script>
 
 <script id="edit-attachment-template" type="text/x-handlebars-template">
-{% verbatim %}
 <form>
   <div class="mb-2"><input type="text" class="form-control form-control-sm" name="filename" required></div>
   <div>
-    <button class="btn btn-sm btn-success save-attachment" type="submit">OK</button>
-    <button class="btn btn-sm btn-outline-secondary cancel">Cancel</button>
+    <button class="btn btn-sm btn-success save-attachment" type="submit">{% trans 'OK' %}</button>
+    <button class="btn btn-sm btn-outline-secondary cancel">{% trans 'Cancel' %}</button>
   </div>
 </form>
```

Comment:
Sure we didn't need the verbatim around this form?

Reply:
No, verbatim is only necessary so that `{{ foo }}` is ignored, because handlebars uses the same syntax. We don't have those here, so it's not necessary.